src/main/java/com/theironyard/entities/Review.java

```diff
@@ -0,0 +1,79 @@
+package com.theironyard.entities;
+
+import javax.persistence.*;
+
+/**
+ * Created by Zach on 10/13/16.
+ */
+@Entity
+@Table(name = "reviews")
+public class Review {
+
+    @Id
+    @GeneratedValue
+    int id;
+
+    @Column(nullable = false)
+    String author;
+
+    @Column(nullable = false)
+    String text;
+
+    @Column(nullable = false)
+    int lecturerId;
```

Comment:
FYI, If the review is tied to the Lecturer shouldn't you be using the @ManyToOne annotation with a Lecture type field? Although the UI does call an individual Lecturer's reviews this assignment does not require that route so the general findAll() works okay.